os/arch/xtensa/include/lx6/chip.h

```diff
@@ -0,0 +1,94 @@
+/******************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+/****************************************************************************
+ * arch/xtensa/include/lx6/chip.h
```

Comment:
In my understanding lx6 is a chip. I think we dont require lx6 chip code..
If at all required, please add a seperate commit for chip code.

Reply:
hi, manoharahk, sorry for my late reply. Xtensa lx is not a chip, it is the enhance technology of xtensa. Xtensa Lx is based on xtensa, using the xtensa ISA instuction set and optional FLIX instructions extensions. If the macro is enabled in the .config, the feature will be inlcuded.  Without Xtensa code, Lx6 could not be used. It's not same to the relationship between armv6-m and armv6-r. So, may i keep it?

Reply:
OK. We can keep it